barretenberg/cpp/src/barretenberg/vm/generated/AvmMini_composer.test.cpp

```diff
@@ -39,25 +39,31 @@ TEST_F(AvmMiniTests, basic)
     trace_builder.add(4, 5, 5);
     trace_builder.add(5, 5, 5);
     trace_builder.add(5, 6, 7);
+    trace_builder.sub(7, 6, 8);
```

Comment:
It would be nice to have comments that reveal the intermediate values, its not clear if a divide by 0 or anything is happeneing in these tests

Reply:
I will work on writing unit tests and will try to document this in a more proper way. I will nevertheless mention the division by zero here.